### PR TITLE
rt-audio 4.1.1

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/rt-audio.rb [hunk 1]
```diff
@@ -1,15 +1,26 @@
-require 'formula'
-
 class RtAudio < Formula
-  homepage 'http://www.music.mcgill.ca/~gary/rtaudio/'
-  url 'http://www.music.mcgill.ca/~gary/rtaudio/release/rtaudio-4.0.12.tar.gz'
-  sha1 'a9a4783183e47f07cff0c62d3882ee95b4eefc4d'
+  homepage "http://www.music.mcgill.ca/~gary/rtaudio/"
+  url "http://www.music.mcgill.ca/~gary/rtaudio/release/rtaudio-4.1.1.tar.gz"
```
Comment (vszakats) [inline Library/Formula/rt-audio.rb]: Would you update above two URLs to use `https`?

Comment (dunn) [inline Library/Formula/rt-audio.rb]: No problem!

**Comment by dunn** [comment]
Bot error, I think, on ML?

File: Library/Formula/rt-audio.rb [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      lib.install "librtaudio_static.a", "librtaudio.dylib"
+    end
+    include.install "RtAudio.h", Dir["include/*"]
+    prefix.install "contrib", "tests"
+    doc.install Dir["doc/*"]
+  end
+
+  test do
```
Comment (asparagui) [inline Library/Formula/rt-audio.rb]: :smiley_cat:

**Comment by xu-cheng** [comment]
Thanks @dunn

**Pull request closed without merging**